### PR TITLE
DOC: update deadlines for 2022

### PR DESCRIPTION
**Pull request by deniederhut**

### CODE REVIEW

**Comment by scoobies** [comment]
Hi!, I'm @scoobies, a friendly bot.

Type ```@scoobies help``` to discover how I can help you.

File: README.md [hunk 1]
```diff
@@ -120,9 +120,9 @@ In addition to the following list, we break up the deadlines in the respective d
 - May 27: 1st Draft for Submission
 - May 27–June 23: Open Review Period
 - May 27: Reviewers Assigned
-- June 9: Initial Complete Review
-- June 23: Final Recommendation and Comprehensive Review Deadlines
-- June 29: Final Author Revision Deadline
+- June 16: Initial Complete Review
+- June 30: Final Recommendation and Comprehensive Review Deadlines
+- July 4: Final Author Revision Deadline
```
Comment (mepa) [inline README.md:125]: This line can also be updated in the "Author deadlines" section below.

**Review by mepa — APPROVED** [review]
Thanks, Dillon!

File: README.md (hunk 1 above)
Comment (deniederhut) [reply README.md:125]: Good catch!

**Pull request merged**